view/frontend/web/template/payment/amwal-payment/form.html

```diff
@@ -71,7 +71,7 @@
         </div>
         <div class="actions-toolbar">
             <div class="primary">
-                <div id="amwal-place-order-button" data-trigger-context="regular-checkout" data-locale="ar">
+                <div id="amwal-place-order-button" data-trigger-context="regular-checkout" data-locale="ar" data-scope-code="">
```

Comment:
Are we using this new scope-code anywhere?

Reply:
Yes we use it